.travis.yml

```diff
@@ -1,9 +1,22 @@
 language: csharp
-sudo: false
-mono:
-  - beta
+sudo: required
+dist: trusty
+addons:
+  apt:
+    packages:
+    - gettext
```

Comment:
Likes it does yah? Or are you suggesting http://docs.asp.net/en/latest/getting-started/installing-on-linux.html#install-the-net-execution-environment-dnx is out-of-date?